cypress/integration/browser/EmptyEventsStreams.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Empty event stream
+
+  I want to check that browser view hide needless elements when I select query with empty event stream
+
+  Background:
+    Given I open a Data Explorer application
+
+  @smoke-test
+  Scenario: Needless UI elements are hidden and edit button is disabled
```

Comment:
I suggest update description to something like this: User is not allowed to edit saved query
The rest test conditions are described in **And** cases.